### ISSUE TITLE
Writing manuscript methods with tedana steps

### ISSUE THREAD

**Issue by eurunuela** [ME-ICA/tedana#975]
### Summary

Yesterday I saw a presentation about [neuromaps](https://netneurolab.github.io/neuromaps/) given by Bratislav Misic where he mentioned neuromaps generates the methods section every time it is run.

I wanted to suggest that now we are giving tedana users the option to use their own decision tree recipes, it could be a good idea to generate a methods section based on the different criteria used. This would include both PCA and ICA criteria used to select the amount of components and their classification.

I wanted to know what everyone involved in tedana thinks about this, before opening a PR. I also think it could be a good way to start contributing for those who want to dive into the code for the first time.

**Comment by tsalo** [comment]
How would this differ from the decision tree documentation in #959?

**Comment by eurunuela** [comment]
I was thinking it would write a couple of paragraphs that users could copy and paste into their manuscripts. They could go either into a `.md` file or a new section of the reports.

**Comment by tsalo** [comment]
Right, but isn't that what I'm doing in #959? Is there more/different info that needs to be added?

**Comment by eurunuela** [comment]
You're right. I'm sorry I missed that part of the PR Taylor. I had not checked the reports before.

Re: PCA, I guess we could explore the possibility in another issue and close this one?

**Comment by tsalo** [comment]
I'm happy to just repurpose this issue to PCA documentation. If there's anything else for the decision trees that should go in the reports that isn't covered in my PR, I also think this issue could be useful.
